doc/contributing/reviewing-contributions.xml

```diff
@@ -493,6 +493,120 @@
 </screen>
   </example>
  </section>
+ <section xml:id="reviewing-contributions-maintainers">
+  <title>Individual Maintainer List</title>
+
+  <para>
+   When adding users to <filename>maintainers/maintainer-list.nix</filename>, the following checks should be performed:
+  </para>
+
+  <itemizedlist>
+   <listitem>
+    <para>
+     If the user has specified a GPG key, verify that the <literal>longkeyid</literal> field matches the <literal>fingerprint</literal>.
+    </para>
+    <example>
+     <title>Validating a maintainer's GPG key</title>
+     <para>
+      First, validate that the commit adding the maintainer is signed by the key the maintainer listed. On GitHub, click the green "Verified" text next to the commit and ensure the "GPG key ID" listed matches the <literal>longkeyid</literal>.
+     </para>
+     <para>
+      If the commit is not signed or it is signed by a different user, ask them to either recommit using that key or to remove their key information.
+     </para>
+     <para>
+      Then validate the fingerprint matches their longkeyid. Given a maintainer entry like this:
+     </para>
+<programlisting language="nix">
+{
+  example = {
+    email = "user@example.com";
+    name = "Example User";
+    keys = [{
+      longkeyid = "rsa8192/0xF04F7A19AAA63AFE";
```

Comment:
key ids are only fingerprints truncated to 64 bits (note here how it's the tail end of the next line), which is few enough to no longer be cryptographically secure. the only reason that key ids ever existed (other than a mistake in the openpgp spec) is to have a handle that is "easier to handle for humans".

unless we make the assumption that humans should ever handle such data manually, there is no practical use case for key ids that holds up either cryptographically or ux-wise. we should deprecate them where possible, or at least not further institutionalize them.

Reply:
Can you cite a reference w.r.t. longkeyids being cryptographically insecure? I was under the impression they were secure, and the default shortkey ID was insecure.

I would like it if you opened a PR removing the longkeyid from the maintainer list with that reference. That said, this change is out of scope for this particular PR, since it is documenting the process as it is today.

Given longkeyid being insecure, this PR will need to be updated to validate the signature on the commandline, since GitHub only shows the longkey.

Reply:
https://dkg.fifthhorseman.net/blog/openpgp-key-ids-are-not-useful.html let's ditch 'em

Reply:
I'll make a PR :+1: